js/containers/app.tsx

```diff
@@ -12,13 +13,17 @@ interface IProps {
 export class App extends React.PureComponent<IProps> {
   render() {
     const { viewType } = this.props;
-    return (
-      viewType === PORTAL_DASHBOARD
-        ? <PortalDashboardApp />
-        : viewType === DASHBOARD
-          ? <DashboardApp />
-          : <ReportApp />
-    );
+    switch (viewType) {
+      case DASHBOARD:
+        return <DashboardApp />;
+      case PORTAL_DASHBOARD:
+        return <PortalDashboardApp />;
+      case IFRAME_STANDALONE:
+        return <IframeStandaloneApp />;
```

Comment:
This is untested, so I'd guess there isn't a basic cypress test to open the iframe standalone.